scripts/nn/layers/softmax_loss.dml

```diff
@@ -0,0 +1,108 @@
+#-------------------------------------------------------------
```

Comment:
we already have softmax and cross_entropy_loss layers.  let's have Caffe2DML simply use those two layers, rather than addin this new wrapper.